apps/cookbook/src/app/examples/accordion-example/examples/events.ts

```diff
@@ -0,0 +1,32 @@
+import { Component } from '@angular/core';
+
+import { ToastConfig, ToastController } from '@kirbydesign/designsystem';
+const config = {
+  selector: 'cookbook-accordion-events-example',
+  template: `<kirby-accordion>
+      <kirby-accordion-item title="Title for accordion item 1" isExpanded={true}  (toggle)="onToggleChange($event)">
+        Toggle to see "toggle" event in action
+      </kirby-accordion-item>`,
+  codeSnippet: `onToggleChange(toggle: boolean) {
+        ...
+      }`,
```

Comment:
Consider readability:
```suggestion
  codeSnippet: `onToggleChange(toggle: boolean) {
  ...
}`,
```